docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -20,85 +20,124 @@ zkApps can now be deployed to Berkeley Testnet.
+- MacOs [Homebrew](https://brew.sh/)
 
-:::
+- Windows [Chocolatey](https://chocolatey.org/)
+
+- Linux (apt, yum, and others)
+
+    As recommended by the Node.js project, you might need to install a recent Node.js version using NodeSource binary distributions: [Debian](https://github.com/nodesource/distributions#debinstall), [rpm](https://github.com/nodesource/distributions#rpminstall).
+
+## Start a project
+
+Now that you have the Mina zkApp CLI installed, you can start with an example or start your own project.
+
+### Option A: Start with an example (recommended)
+
+Examples are based on the standard project structure and provide additional files in the `/src` directory.
+
+1. Create the sudoku example project: `zk example sudoku`
+
+  The created project includes the example files (the smart contract) in the project's `src/` directory. 
+  
+  To see the files that were created, change to the `sudoku` directory and run the `ls` command or open the directory in a code editor, such as VS Code.
+
+1. Run tests and see the tests pass: `npm run test`  
+  
+  To rerun tests automatically when you save changes to your code, run the tests in watch mode with `npm run testw`.
+
+1. Build the example: `npm run build`
+
+  Compile your TypeScript into JavaScript in the project `/build` directory.
+
+1. Configure your zkApp: `zk config`
+
+  The command prompts guide you in adding a network alias to your project `config.json` file. 
+  
+  For Berkeley Testnet, use:
+  
+    - name: `berkeley`
+    - Mina GraphQL API URL: `https://proxy.berkeley.minaexplorer.com/graphql`
+    - transaction fee: `0.1`
+
+1. Fund your fee payer account.
+
+  Follow the prompts to request tMina. 
+
+1. Deploy to Testnet: `zk deploy`
+
+  Follow the prompts. For details, see [how to deploy a zkApp](how-to-deploy-a-zkapp).
+
+### Option B: Start your own project
+
+1. Create your own project: `zk project <myproj>`
+
+    The created project includes the files (the smart contract) in the project's `src/` directory. 
+
+    To see the files that were created, change to the project (whatever you called `<myproj>`) directory and run the `ls` command or open the directory in a code editor, such as VS Code.
+
+1. Run tests and see the tests pass: `npm run test`  
+  
+    To rerun tests automatically when you save changes to your code, run the tests in watch mode with `npm run testw`.
+
+1. Build the example: `npm run build`
+
+    Compile your TypeScript into JavaScript in the project `/build` directory.
+
+1. Configure your zkApp: `zk config`
+
+    The command prompts guide you in adding a network alias to your project `config.json` file. 
```

Comment:
Same comment as before:
Change `network alias` to  `deploy alias`. 
 
Note that the deploy alias can be anything a user wants.